recipes/xitorch/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "xitorch" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xitorch/xitorch/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 763d1935f5944589da3cb80a84d2bbdc9f5db193ff9e6893581eff1460e3f54e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - numpy >=1.8.2
+    - python >=3.7
+    - scipy >=1.1.0
+    - pytorch
+
+test:
+  imports:
+    - {{ name }}
+  commands:
+    - pip check
+    - pytest -vv --pyargs {{ name }} -k 'not test_equil['
```

Comment:
```suggestion
```
The main purpose of the test section in a conda recipe is to test *packaging* not correctness. Since the correctness tests are unstable, don't do them. Pip check should be sufficient.